example/config/garden.yaml

```
@@ -17,6 +17,9 @@ providers:       # contains information about known providers
     #   skip:
     #     enabled: true
     #     justification: "the whole rule is accepted for ... reasons"
+    - ruleID: "2007"
+      args:
+        minPodSecurityLevel: baseline
```

Should we comment this in order to note that this field is optional 🤔 
```suggestion
      # args:
      #   minPodSecurityStandardsProfile: baseline # if set it will indicate the min Pod Security Standards profile that is allowed. Possible values are "privileged", "baseline" and "restricted".  
```

---

Now that I think of it we can comment out the whole rule.